src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -190,6 +192,26 @@ function adjustStudentsPanelView(panelHeading, panelCollapse, handsontableInstan
     handsontableInstance.render();
 }
 
+/**
+ * Adds all error messages returned from the backend into a map.
+ */
+function addEnrollErrorMessages(enrollErrorLines) {
+    // Updates any error messages to process later
+    if (!jQuery.isEmptyObject(enrollErrorLines)) {
+        Object.keys(enrollErrorLines).forEach(key => (
+                enrollErrorMessagesMap.set(key, enrollErrorLines[key])
```

Comment:
Why creating the `enrollErrorMessageMap` object while you can just use the `enrollErrorLines` object directly?

Reply:
I wanted `enrollErrorLines` to be in the form of a `Map`, thus this choice. I did some refactoring in the later commits to convert the object directly into a `Map`, i.e. https://github.com/TEAMMATES/teammates/blob/4c7a547a1c6b10e5b3450b2e6c9b951fb8c2d13e/src/main/webapp/dev/js/main/instructorCourseEnrollPage.js#L263